src/Common/src/Interop/Windows/kernel32/Interop.FILE_INFO_BY_HANDLE_CLASS.cs

```diff
@@ -0,0 +1,152 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        internal enum FILE_INFO_BY_HANDLE_CLASS : uint
+        {
+            // Up to FileRemoteProtocolInfo available in Windows 7
+
+            /// <summary>
+            /// Returns basic information (timestamps and attributes).
+            /// </summary>
+            /// <remarks>
+            /// Thunks to NtQueryInformationFile & FileBasicInformation.
```

Comment:
`&` needs to be escaped in xml as `&amp;` (applies throughout these xml doc comments)

Reply:
Changing